<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyHUD</class>
 <widget class="QWidget" name="KeyHUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keys</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KeyComboBox" name="up_comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KeyComboBox" name="down_comboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KeyComboBox" name="left_comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KeyComboBox" name="right_comboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Shoot</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KeyComboBox" name="shoot_comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">keycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
